src/_tests/example_layouts/__snapshots__/_wireframe.test.tsx-simple-wireframe-with-background-color.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
@orta Sorry it took so long to update this PR. Is this what you wanted?

Reply:
I don't think so, it feels like the  crux of the matter is that we're building two different-ish things:

* You're making a 💯 SVG renderer for a React Component
* I'm aiming something that shows the layout of my React components

For example, in this PR, this SVG went from showing something - to showing nothing:

<img width="1156" alt="screen shot 2017-12-06 at 9 32 29 am" src="https://user-images.githubusercontent.com/49038/33666631-952292d4-da68-11e7-8543-89c604c23c41.png">

Which is basically what will happen to all my tests if I upgrade to this version of jest-snapshots-svg 😺 

I **don't** think these two aims are divergent goals at all, and perhaps the multi-package route is  the right option option: where there is a package for react-to-svg rendering, and the testing lib which can fill in the gaps like "this component has no border or BG, so add a default". 

In the mean time, I'm happy to figure a way to get this in somehow, it's been a long PR and it'd be good to get it in

Reply:
I've updated it again. I think this was just a few tests wanting the `{ wireframe: true }` option. Let me know if this works for your use-case! (Sorry again about the delay)

Reply:
No pressure, this looks 👍